yt_dlp/extractor/masters.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import url_or_none
+
+
+class MastersIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?masters\.com/en_US/watch/[\d]{4}-[\d]{2}-[\d]{2}/(?P<id>[0-9]+)/(.*)\.html'
+    _TESTS = [{
+        'url': 'https://www.masters.com/en_US/watch/2022-04-07/16493755593805191/sungjae_im_thursday_interview_2022.html',
```

Comment:
is the date in URL the upload date? If so, extract it

Reply:
Yes. will return that as well